catalog/ACA/ACA_reef_habitat_v1_0.jsonnet

```diff
@@ -73,7 +75,7 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
   keywords: [
     'coral',
     'ocean',
-    'planet',
+    'sentinel2_derived',
```

Comment:
Please sort the keywords

Issue('ACA/reef_habitat/v1_0', 'ACA/ACA_reef_habitat_v1_0.json', 'keywords', '"keywords" must be sorted', IssueLevel.error)